presto-spark-base/src/main/java/com/facebook/presto/spark/execution/property/NativeExecutionVeloxConfig.java

```diff
@@ -29,12 +29,20 @@ public class NativeExecutionVeloxConfig
     private static final String AGGREGATION_SPILL_ENABLED = "aggregation_spill_enabled";
     private static final String JOIN_SPILL_ENABLED = "join_spill_enabled";
     private static final String ORDER_BY_SPILL_ENABLED = "order_by_spill_enabled";
+    private static final String MAX_SPILL_BYTES = "max_spill_bytes";
+    private static final String MAX_SPILL_RUN_ROWS = "max_spill_run_rows";
 
     private boolean codegenEnabled;
     private boolean spillEnabled = true;
     private boolean aggregationSpillEnabled = true;
     private boolean joinSpillEnabled = true;
     private boolean orderBySpillEnabled = true;
+    // Velox default value is 100G, as it is designed for Presto cluster
+    // use-case. But for presto-on-spark, we do not have any limit
+    private Long maxSpillBytes = 0L;
+    // Same as above comments. We do not need to set limit on
+    // how many rows we can spill
+    private Long maxSpillRunRows = 0L;
```

Comment:
Let's keep this to avoid memory allocation error during the spill. We can tune this a bit later if create too many spill files. Thanks!

Reply:
Sounds good. Updated.